main.tf

```diff
@@ -181,6 +187,11 @@ resource "google_alloydb_instance" "read_pool" {
     cpu_count = each.value.machine_cpu_count
   }
 
+  client_connection_config {
```

Comment:
Make this also a dynamic block like [the one in ](https://github.com/GoogleCloudPlatform/terraform-google-alloy-db/pull/59/files#r1466969112)

Reply:
Fixed but required_connectors order is bit different than suggested.